src/main/java/com/box/sdk/BoxFile.java

```diff
@@ -462,6 +480,52 @@ public URL getPreviewLink() {
         return info.getPreviewLink();
     }
 
+
+    /**
+     * Retrieves a thumbnail, or smaller image representation, of this file. Sizes of 32x32, 64x64, 128x128,
+     * and 256x256 can be returned in the .png format and sizes of 32x32, 94x94, 160x160, and 320x320 can be returned
+     * in the .jpg format.
+     * @param fileType      either PNG of JPG
+     * @param minWidth      minimum width
+     * @param minHeight     minimum height
+     * @param maxWidth      maximum width
+     * @param maxHeight     maximum height
+     * @return the byte array of the thumbnail image
+     */
+    public byte[] getThumbnail(ThumbnailFileType fileType, int minWidth, int minHeight, int maxWidth, int maxHeight) {
+        QueryStringBuilder builder = new QueryStringBuilder();
+        builder.appendParam("min_width", minWidth);
+        builder.appendParam("min_height", minHeight);
+        builder.appendParam("max_width", maxWidth);
+        builder.appendParam("max_height", maxHeight);
+
+        URLTemplate template;
+        if (fileType == ThumbnailFileType.PNG) {
+            template = GET_THUMBNAIL_PNG_TEMPLATE;
+        } else {
```

Comment:
I'd change this to an else-if to explicitly check for `ThumbnailFileType.JPG` to avoid bugs if a new filetype is ever added.